### ISSUE TITLE
Issue with "Right shift (>>)": (comment in JavaScript Demo: Expressions - Right shift operator has problem)

### ISSUE THREAD

**Issue by blateyang** [mdn/content#6392]
MDN URL: https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Operators/Right_shift

#### What information was incorrect, unhelpful, or incomplete?
Hi, I found the comment of `console.log(c >> b); ` in JavaScript Demo: Expressions - Right shift operator as follows has problem

#### Specific section or headline?

```
const a = 5;          //  00000000000000000000000000000101
const b = 2;          //  00000000000000000000000000000010
const c = -5;         // -00000000000000000000000000000101

console.log(a >> b);  //  00000000000000000000000000000001
// expected output: 1

console.log(c >> b);  // -00000000000000001111111111111110
// expected output: -2
```
#### What did you expect to see?
console.log(c >> b);  // -00000000000000000000000000000010

<details>
<summary>MDN Content page report details</summary>

* Folder: `zh-cn/web/javascript/reference/operators/right_shift`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Operators/Right_shift
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/web/javascript/reference/operators/right_shift/index.html
* Last commit: https://github.com/mdn/translated-content/commit/52ccdf1f0a1cee3f0659ac78e81c79dae152399b
* Document last modified: 2021-06-23T00:34:06.000Z

</details>

**Comment by sideshowbarker** [comment]
@mdn/yari-content-zh

**Comment by SaurabhCha** [comment]
Hi @sideshowbarker, Can I fix this comment ?

**Comment by sideshowbarker** [comment]
@SaurabhCha this is in the zh-CN (Chinese-localized) version at https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Operators/Right_shift — so the fix would need to be made in the https://github.com/mdn/translated-content/ repo

**Comment by SaurabhCha** [comment]
@sideshowbarker Thanks for let me know, I didn't know about that. I will look for something else.

**Comment by t7yang** [comment]
> 
> 
> @SaurabhCha this is in the zh-CN (Chinese-localized) version at https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Operators/Right_shift — so the fix would need to be made in the https://github.com/mdn/translated-content/ repo

@sideshowbarker Not really, this is a interactive example which embed by `EmbedInteractiveExample("pages/js/expressions-right-shift.html`.

The original file is here:
https://github.com/mdn/interactive-examples/blob/fd8d9cdb45/live-examples/js-examples/expressions/expressions-right-shift.js

So, this issue (if it really an issue) should close or transfer to https://github.com/mdn/interactive-examples.
